genestack_client/data_importer.py

```diff
@@ -590,6 +590,37 @@ def create_mapped_reads_count(self,
         url and metainfo.add_external_link(BioMetaKeys.DATA_LINK, url)
         return self.__invoke_loader(parent, 'mappedReadCounts', metainfo)
 
+    def create_gene_list(self,
+                                  parent=None,
+                                  name=None,
+                                  url=None,
+                                  organism=None,
+                                  metainfo=None):
```

Comment:
This indenting style is too weird even for us.
Please format signature like `create_sequencing_assay()` (preferred) or at least like `create_bam()`.